zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/web/rest/dto/ColumnResponse.kt

```diff
@@ -16,24 +16,32 @@
 
 package com.ritense.documentenapi.web.rest.dto
 
+import com.ritense.documentenapi.domain.ColumnDefaultSort
 import com.ritense.documentenapi.domain.DocumentenApiColumn
 import com.ritense.documentenapi.domain.DocumentenApiColumnId
 import com.ritense.documentenapi.domain.DocumentenApiColumnKey
 
-data class ConfiguredColumnDto(
+data class ColumnResponse(
     val key: String,
-    val enabled: Boolean,
+    val sortable: Boolean,
+    val filterable: Boolean,
+    val defaultSort: String?,
 ) {
-    fun toEntity(caseDefinitionName: String, order: Int = 0): DocumentenApiColumn = DocumentenApiColumn(
-        id = DocumentenApiColumnId(caseDefinitionName, DocumentenApiColumnKey.valueOf(key.uppercase())),
-        order = order,
-        enabled = enabled
-    )
+    fun toEntity(caseDefinitionName: String, order: Int = 0): DocumentenApiColumn {
+        val keyEnum = DocumentenApiColumnKey.valueOf(key.uppercase())
```

Comment:
technically we don't need the `key.uppercase()` right?

Reply:
technically its not needed, no